### PR TITLE
cirrus: Run extended test feature_pruning

### PR DESCRIPTION
**Pull request by maflcko**
Currently not run in this repo, but in https://cirrus-ci.com/github/MarcoFalke/bitcoin

https://cirrus-ci.com/github/MarcoFalke/bitcoin/1903-ciExt

### CODE REVIEW

**Comment by practicalswift** [comment]
Concept ACK

File: .cirrus.yml [hunk 1]
```diff
@@ -2,15 +2,16 @@ task:
...
   env:
     MAKEJOBS: "-j3"
     CONFIGURE_OPTS: "--disable-dependency-tracking"
     GOAL: "install"
     CCACHE_SIZE: "200M"
     CCACHE_COMPRESS: 1
     CCACHE_DIR: "/tmp/ccache_dir"
+  ccache_cache:
+    folder: "/tmp/ccache_dir"
+  timeout_in: 80m
```
Comment (fkorotkov) [inline .cirrus.yml]: If you are hitting the time out limit but your build can benefit from more CPUs, you can try to increase VM resources to the maximum allowed:

```yaml
freebsd_instance:
    image: freebsd-12-0-release-amd64
    cpu: 8
    memory: 24G
```

**Comment by fanquake** [comment]
utACK 394fc4d

Looks like the last full run on a FreeBSD VM (of master) completed successfully.

**Comment by MarcoFalke** [comment]
I had to exclude feature_dbcrash, but at least we are running feature_pruning on freebsd now and the build is faster. Going to merge this.

**Pull request merged**